src/main/java/io/airbyte/testingtool/scenario/action/ActionFactory.java

```diff
@@ -54,20 +72,39 @@ private static <T extends Instance> T getInstance(String instanceName,
     return clazz.cast(scenarioInstanceNameToInstanceMap.get(instanceName));
   }
 
-  private static List<Instance> getRequiredInstances(ScenarioConfigAction config, Map<String, Instance> scenarioInstanceNameToInstanceMap) {
+  private static List<Instance> getRequiredInstances(ScenarioConfigAction config,
+      Map<String, Instance> scenarioInstanceNameToInstanceMap) {
     return scenarioInstanceNameToInstanceMap.entrySet().stream()
-        .filter(nameInstanceEntry -> config.getRequiredInstances().contains(nameInstanceEntry.getKey())).map(
+        .filter(
+            nameInstanceEntry -> config.getRequiredInstances().contains(nameInstanceEntry.getKey()))
+        .map(
             Entry::getValue).toList();
   }
 
-  private static ActionConnectToAirbyteAPI getActionConnectToAirbyteAPI(int order, ScenarioConfigAction config,
+  private static String getParamByType(ScenarioConfigAction config,
```

Comment:
Please move parameter type input to the first position in order to be inline with similar method for intances